openqa-trigger-bisect-jobs

```diff
@@ -299,10 +300,17 @@ def main(args):
             ]
         )
 
-        out = openqa_clone(
-            params,
-            args.dry_run,
-        )
+        try:
+            out = openqa_clone(
+                params,
+                args.dry_run,
+            )
+        except subprocess.SubprocessError as err:
+            p_stderr = err.stderr if err.stderr else "No error putput"
+            log.error(f"{type(err).__name__}: {p_stderr} ")
+            #openqa_comment(base_url, job_id, err_comment, args.dry_run)
```

Comment:
We said that we only want AC1 for the first PR, so you can leave that out.
But it's good to keep that in mind.

Reply:
Can I leave it or add a todo or...?

Reply:
Remove it